src/mame/konami/spy.cpp

```diff
@@ -285,64 +299,21 @@ void spy_state::bankswitch_w(uint8_t data)
 	membank("bank1")->set_entry(bank);
 }
 
-void spy_state::spy_collision(  )
+void spy_state::pmc_run(  )
 {
 #define MAX_SPRITES 64
 #define DEF_NEAR_PLANE 0x6400
 #define NEAR_PLANE_ZOOM 0x0100
 #define FAR_PLANE_ZOOM 0x0000
```

Comment:
Can you replace these with `constexpr` locals while you’re working on this?  They’re easier to deal with.